lib/flight.js

```diff
@@ -6,10 +6,32 @@ const getProp = require('dot-prop').get
 const moment = require('moment-timezone')
 
 const endpoint = 'https://data-live.flightradar24.com/clickhandler/'
+
+// const headers = {
+// 	'User-Agent': 'https://github.com/derhuerst/fetch-flightradar24-flights'
+// }
+
+/// PKO header
+/// Force a rotation of user-agent string, to avoid autmated blocking @FR24 side
+/// Data source: https://www.useragents.me/  i take manually the top 10 
+///////////////////////////////////////////////////////////////////////////////////////////
+const uas= ("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.246"
```

Comment:
In JavaScript, the `(…, …)` notation *does not* denote an array, but merely evaluates to the last expression. Therefore, your code would only ever use the last of the 10 user agents enumerated here.